test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/Utils.scala

```diff
@@ -95,7 +95,7 @@ object Utils {
     val cmdStr = cmd.mkString(" ")
     printMsg(s"Running command: ${cmdStr} with env vars: ${env.mkString(" ")}")
     val proc = Process(cmdStr, None, env: _*)
-    proc.lines_!.foreach(line => println(line))
+    proc.!!
```

Comment:
The problem with `lines_!` was that it would not throw an exception if the program terminate with a non-zero return code?
The thing is with using `!!`, we are also losing the output. Looking at the Scaladoc, it looks like using `lineStream` (`line` is deprecated) would allow to have the output, and get an exception at the end.